buildenv

```diff
@@ -1,14 +1,16 @@
+SHDOC_VERSION="1.0.0"
 export ZOPEN_STABLE_URL="git@github.com:reconquest/shdoc.git"
```

Comment:
Can we use https protocol to avoid requiring a key?

Reply:
don't we have other stable builds that use git@github?

Reply:
I have modified to use https, anyways i dont have any hard rule here.